### DISCUSSION TITLE
Problem running upgrade-run.sh on V2

### DISCUSSION THREAD

**Discussion by the-real-crayboy** [gtxaspec/wz_mini_hacks#353] [Q&A]
I just got wz_mini_hacks running on a V2.
I ran upgrade-run.sh and got as far as 
`Rebooting into UPGRADE MODE`
My camera is now clicking every 5s or so. Solid yellow light.
Should I keep waiting?

I didn't. 

I power cycled it - back to clicking every 5s - Light would be solid for a while then off ( synced to clicks )

Ended up reformatting card/recopying files - camera is running again.

**Comment by bjs-pdx** [comment]
What network adapter are you using?  I can't get the Uctronics to work on a V2...only on V3 or Pan V2.
